project_palnning.txt

```diff
@@ -3,4 +3,7 @@ Phase 1:
 
 	Analyse the requirement
 	Create estimation sheet
+
+Phase 2:
```

Comment:
Why coding included in phase 2? We planned it in phase 3

Reply:
Ok Ill update Phase 2 and push it